src/ReactNativeSVG.web.ts

```diff
@@ -40,7 +39,10 @@ import type {
 } from './lib/extract/types';
 import SvgTouchableMixin from './lib/SvgTouchableMixin';
 import { resolve } from './lib/resolve';
-import { transformsArrayToProps } from './lib/extract/extractTransform';
+import {
+  transformsArrayToProps,
+  TransformsStyleArray,
```

Comment:
If it is a type, please import it separately in `import type` style. Also, if it fails, I guess it also fails on `main` yes? If so, please submit another PR fixing it since it does not seem to be logically part of this PR.